app/controllers/passengers_controller.rb

```diff
@@ -48,8 +47,17 @@ def destroy
   private
 
   def passenger_params
-    params.require(:passenger).permit(:name, :address, :email, :phone,
-      :wheelchair, :active, :permanent, :expiration, :note)
+    permitted_params = params.require(:passenger).permit(:name, :address,
```

Comment:
I might align `permit` with `require` so that you can fit more attributes on a line.